package.json

```diff
@@ -48,7 +48,8 @@
     "terser": "^4.3.9",
     "unistore": "^3.4.1",
     "wicg-inert": "^3.0.1",
-    "workbox-build": "^4.3.1"
+    "workbox-build": "^5.0.0",
+    "workbox-expiration": "^5.0.0"
```

Comment:
`workbox-build` results in a transitive dependency on all the Workbox runtime libraries, so they all end up in `node_modules/`, but I think it's better practice to explicitly list all the dependencies here. (Right now you're only listing `workbox-expiration`, so that would be `workbox-precaching`, `workbox-routing`, etc.)